source/MaterialXCore/Document.cpp

```diff
@@ -357,11 +359,42 @@ void convertParameterToInput(DocumentPtr doc)
                 {
                     // TODO: Determine based on usage whether to make
                     // the input a uniform. 
-                    newInput->setIsUniform(false);
+                    newInput->setIsUniform(true);
                 }
+                anyConverted = true;
+            }
+        }
+    }
+    return anyConverted;
+}
+
+bool Document::convertUniformInputsToParameters()
+{
+    bool anyConverted = false;
+
+    const StringSet uniformTypes = { FILENAME_TYPE_STRING, STRING_TYPE_STRING };
+    const string PARAMETER_CATEGORY_STRING("parameter");
```

Comment:
Seems like you're using this constant more than once (two different functions), should it be declared elsewhere?

Reply:
I'll take a look.